test/data.jl

```diff
@@ -180,6 +180,18 @@ end
     @test eltype(dropmissing!(df).b) == Int
 end
 
+@testset "dropmissing and unique view kwarg test" begin
+    df = DataFrame(rand(3,4))
+    for fun in (dropmissing, unique)
+        @test fun(df) isa DataFrame
+        @test fun(view(df, 1:2, 1:2)) isa DataFrame
+        @test fun(df, view=false) isa DataFrame
+        @test fun(view(df, 1:2, 1:2), view=false) isa DataFrame
+        @test fun(df, view=true) isa SubDataFrame
+        @test fun(view(df, 1:2, 1:2), view=true) isa SubDataFrame
```

Comment:
Also test returned value? Or is that covered elsewhere?

It would also be nice to use `@inferred` when `view=true/false` isn't specified to prevent any regression: it would be easy to remove one of the `@inlined` without realizing why they are here.

Reply:
I have added the test and `@inferred` (I thought it would fail on Julia 1.0, but it passes - which is good)